Data/SqlDataLayer.cs

```diff
@@ -63,63 +63,176 @@ public Staff Read(int staffIdToRead)
                 SqlCommand command = new SqlCommand("Proc_Staff_ReadById", cnn);
                 command.CommandType = CommandType.StoredProcedure;
                 command.Parameters.Add("@EmpId", SqlDbType.VarChar).Value = staffIdToRead;
-                
+
                 using (SqlDataReader reader = command.ExecuteReader())
                 {
-                   
+
                     do
                     {
-                        
+
 
                         while (reader.Read())
```

Comment:
Do we need a while loop here to get a single result?

Reply:
I'll change that,it was a previously used stored procedure with multiple select statements